docusaurus.config.js

```diff
@@ -252,7 +252,7 @@ const config = {
 		  <img src="/img/linkedin_icon.svg" alt=""/>
 		</a>
             </div>
-            <div class="footer-bottom">Copyright © 2023 The Apache
+            <div class="footer-bottom">Copyright © ${new Date().getFullYear()} The Apache
```

Comment:
not sure if we should update the license this way, as above I think we need to leave the start year in place ???

Reply:
Hey @DaanHoogland do you mean from `2023 to current year` ?

Or should we use an earlier starting year from back when CloudStack started ?

Reply:
no, we should use the year this website started. I am not sure if that should be 2012 (the original version) or 2023 (the docusaurus version) But the jist is that there is contents there from al those years that is copyrighted. I am not a lawyer, so I would really not know what is legal and permissable, I just know that removing an original year may enable people to "legally" copy older versions.

Reply:
I think we should be safe to use `2012 - ${new Date().getFullYear()}`, though.